core/src/test/scala/unit/kafka/utils/JaasTestUtils.scala

```diff
@@ -42,15 +40,38 @@ object JaasTestUtils {
     }
   }
 
-  case class JaasSection(contextName: String,
-                         moduleName: String,
-                         debug: Boolean,
-                         entries: Map[String, String]) {
+  case class PlainLoginModule(username: String,
+                              password: String,
+                              debug: Boolean = false,
+                              validUsers: Option[Map[String, String]] = None) {
+    def toJaasModule: JaasModule = {
+      JaasModule(
+        "org.apache.kafka.common.security.plain.PlainLoginModule",
+        debug = debug,
+        entries = Map(
+          "username" -> username,
+          "password" -> password
+        ) ++ validUsers.getOrElse(Map.empty).map(up => ("user_" + up._1 -> up._2))
+      )
+    }
+  }
+
+  case class JaasModule(moduleName: String,
+                        debug: Boolean,
+                        entries: Map[String, String]) {
     override def toString: String = {
-      s"""|$contextName {
-          |  $moduleName required
+      s"""$moduleName required
           |  debug=$debug
           |  ${entries.map { case (k, v) => s"""$k="$v"""" }.mkString("", "\n|  ", ";")}
```

Comment:
Did you really mean to have different indenting between lines? I think it would be nicer if these 3 lines were at the same indentation. We can change `JaasSection.toString` if it's relying on the current behaviour.